### DISCUSSION TITLE
How to make this kind of badge ?

### DISCUSSION THREAD

**Discussion by SirMishaa** [badges/shields#6875] [Q&A] [answered]
Hello, i've made this a long time ago : 

![image](https://user-images.githubusercontent.com/42717232/128527724-93620904-3740-4587-91dc-2c0b383c8470.png)

But I would like to remove the nestjs badge and to add the Rust badge! 

But, unfortunately I can't make this kind of badge anymore and I can't find much on the doc 
I must be a jerk, but how can I make a badge like this for Rust ? 

In advance, thank you

**Comment by chris48s** [comment, marked as answer]
https://img.shields.io/badge/rust-black.svg?&style=for-the-badge&logo=rust&logoColor=white ?

**Reply by SirMishaa** [reply]
Oh, alright, I understand, thanks!
